minecraft_servers/hielkemapsnetwork/manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+  "server_name": "hielkemapsnetwork",
+  "nice_name": "Hielke Maps Network",
+  "direct_ip": "play.hielkemaps.com",
+  "server_wildcards": [
+    "play.hielkemaps.com"
+  ],
+  "social": {
+    "web": "https://hielkemaps.com",
+    "twitter": "HielkeMaps",
+    "discord": "https://discord.com/invite/6RzKxbH",
+    "youtube": "https://www.youtube.com/@HielkeMinecraft"
+  },
+  "gamemodes": {
+    "parkour": {
+      "name": "Parkour"

```

Comment:
Gamemodes need also the `color` property. See https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#gamemode-object